articles/connections/database/mysql.md

```diff
@@ -87,6 +87,8 @@ function login(email, password, callback) {
 
 This script connects to a **MySQL** database and executes a query to retrieve the first user with `email == user.email`. With the `bcrypt.compareSync` method, it then validates that the passwords match, and if successful, returns an object containing the user profile information including `id`, `nickname`, and `email`. This script assumes that you have a `users` table containing these columns. You can tweak this script in the editor to adjust it to your own requirements.
 
+You can also [cache connections](/rules/current#cache-expensive-resources) such that it will survive individual execution.
```

Comment:
No need to add the version, the link works like that: `[cache connections](/rules#cache-expensive-resources)`

That way if the versions change, we don't have to go over the docs and update all these links.